curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-an-ordered-list.english.md

```diff
@@ -42,6 +42,10 @@ tests:
     testString: assert(code.match(/<\/ol>/g) && code.match(/<\/ol>/g).length === code.match(/<ol>/g).length, 'Make sure your <code>ol</code> element has a closing tag.');
   - text: Make sure your <code>li</code> element has a closing tag.
     testString: assert(code.match(/<\/li>/g) && code.match(/<li>/g) && code.match(/<\/li>/g).length === code.match(/<li>/g).length, 'Make sure your <code>li</code> element has a closing tag.');
+  - text: The <code>li</code> elements in your unordered list should not be empty.
+    testString: $('ul li').each((i, val) => assert(!!val.textContent.replace(/\s/g, '').length, 'Your <code>li</code> elements in your unordered list should not be empty.'));
+  - text: The <code>li</code> elements in your ordered list should not be empty.
+    testString: $('ol li').each((i, val) => assert(!!val.textContent.replace(/\s/g, '').length, 'Your <code>li</code> elements in your unordered list should not be empty.'));
```

Comment:
Likewise here and it should be `ordered list` instead of `unordered list`.

Reply:
@ezioda004 Look at the other tests above.  They have tests for both ordered and unordered lists.

Reply:
@RandellDawson Yes ofcourse but in this case the **text** and the *testString* don't match.
>**text**: The <code>li</code> elements in your *ordered* list should not be empty.
>**testString**: Your <code>li</code> elements in your *unordered* list should not be empty.

Reply:
@ezioda004 - I assume your are talking about the last text and testString.   I was focused on the code and not the `text`.  Great catch!  That is why it is always good to have multiple reviews on these challenges.